db/warehouse/migrate/20250708132033_add_ce_custom_referral_statuses.rb

```diff
@@ -0,0 +1,26 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class AddCeCustomReferralStatuses < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ce_custom_referral_statuses do |t|
+      t.string :key, null: false, index: { unique: true }
```

Comment:
probably should be unique on [key,data_source_id]?

Reply:
https://github.com/greenriver/hmis-warehouse/pull/5556/commits/c140a28fc1cde8f3e7da49e9a3032b74c8a2a520